src/_components/reward/Reward.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useState } from 'react'
+
+import { Gift } from '../../_assets/images';
+import { TReward } from '../../shop/_models/reward';
+import { Points } from '../points/Points';
+import Typography from '../typography/Typography';
+import { InAppRewardLabel } from './InAppRewardLabel';
+import * as Styled from './style';
+
+type TProps = {
+  mode: 'list' | 'tile',
```

Comment:
misschien onnozel maar list en tile zeggen me niet zoveel

Reply:
Voorkeur voor iets anders? Een tile is nu eenmaal een tile, die list zou eventueel iets anders kunnen zijn, suggesties welkom :D